decentraland/screens/MyAccountScreen.js

```diff
@@ -1,53 +1,73 @@
 import React from "react";
 import MyAccount from "@presentational/MyAccount";
 import { connect } from "react-redux";
-import AccountCreationActions from "@constants/AccountCreationActions";
-import ActionStatus from "../constants/ActionStatus";
+import {
+  FUNDING_WITH_ETH,
+  FUNDING_WITH_MANA,
+  APPROVING_MARKETPLACE,
+} from "@constants/AccountCreationStatus";
+import { MISSING, DONE } from "@constants/ActionStatus";
 import PropTypes from "prop-types";
 
+const StepsWithAction = [
+  {
+    creationStatus: FUNDING_WITH_ETH,
+    name: "Funded with ETH",
+    percentage: 0.25,
+  },
+  {
+    creationStatus: FUNDING_WITH_MANA,
+    name: "Funded with MANA tokens",
+    percentage: 0.25,
+  },
+  {
+    creationStatus: APPROVING_MARKETPLACE,
+    name: "Linked to marketplace",
+    percentage: 0.25,
+  },
+];
+
 export class MyAccountScreen extends React.Component {
   render() {
     const { accountInfo } = this.props;
-    const creationActions = [];
-    creationActions.push({
+    const { creationActions, account } = accountInfo;
+    const isAccountCreated = account !== null;
+
+    const creationSteps = [];
+
+    creationSteps.push({
       name: "Account created",
-      action: null,
-      status: !accountInfo.account ? ActionStatus.MISSING : ActionStatus.DONE,
+      creationStatus: null,
```

Comment:
What does this get used for?

Reply:
Or put another way, why push things onto `creationSteps` in two spots? (This happens a few lines up and then lower down in this file.)

Reply:
During account generation, no data is being saved on `accountInfo.creationActions` and the loop is looking into this object to determine if it was done or not.
Note: It seems odd to me too, I've just refactored the code keeping the pre-existent logic.

Reply:
Okay, thanks. You haven't pushed up that change yet, right?

Reply:
Just done.

Reply:
Resolving this